docs/usage.md

```diff
@@ -1,46 +1,45 @@
 # Basic Usage
 
-`jrnl` has two modes: **composing** and **viewing**. Basically, whenever
-you _don't_ supply any arguments that start
-with a dash or double-dash, you're in composing mode, meaning you can
-write your entry on the command line or an editor of your choice.
-
-We intentionally break a convention on command line arguments: all
-arguments starting with a _single dash_
-will _filter_ your journal before viewing
-it, and can be combined arbitrarily. Arguments with a
-_double dash_ will control how your journal
-is displayed or exported and are mutually exclusive (ie. you can only
-specify one way to display or export your journal at a time).
+`jrnl` has two modes: **composing** and **viewing**. Whenever you don't enter
+any arguments that start with a dash (`-`) or double-dash (`--`), you're in
+composing mode, meaning that you can write your entry on the command line.
+
+We intentionally break a convention on command line arguments: all arguments
+starting with a _single dash_ (`-`) will _filter_ your journal before viewing
+it. Filter arguments can be combined arbitrarily. Arguments with a _double dash_
+(`--`) will _control_ how your journal is displayed or exported. Control
+arguments are mutually exclusive (i.e., you can only specify one way to display
+or export your journal at a time).
 
 ## Listing Journals
 
-You can list the journals accessible by jrnl
+You can list the journals accessible by `jrnl`:
 
 ```sh
 jrnl -ls
```

Comment:
Ugh, we really need to fix the `-ls` command. It doesn't fit our own stated paradigm of single-dash/double-dash commands.

Reply:
Seems to me like the writing/reading/filtering commands use a single dash and the export/import/edit/setup commands use double dashes. Why not `jrnl --list` or something like that?

Reply:
> Seems to me like the writing/reading/filtering commands use a single dash and the export/import/edit/setup commands use double dashes.

Duh. It even says that in the file we're discussing. 🤦